ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -882,8 +887,10 @@ def arrays_to_numpy(x):
                 frontend_ret_np_flat = [ivy.to_numpy(x) for x in frontend_ret_flat]
             # unset frontend framework from backend
             ivy.previous_backend()
+            ivy.unset_backend()
```

Comment:
I think this is old changes from old commit, please also remove this (repetition with the existing code - name change)